### ISSUE TITLE
pgvector-python client support for sparse vectors.

### ISSUE THREAD

**Issue by anugrahastephen** [pgvector/pgvector-python#71]
Does PGVector python client supports sparse vector embeddings storage in postgresDB like Splade?

**Comment by ankane** [comment]
Hi @anugrahastephen, it's available on master, but isn't released yet (still making adjustments). I also added [an example for sparse search](https://github.com/pgvector/pgvector-python/blob/master/examples/sparse_search.py).

**Comment by anugrahastephen** [comment]
Hi @ankane , thank you for your quick response. Hope you release the feature soon. 
Thanks for the project, it has been a great experience working with it.

**Comment by ankane** [comment]
Just released 0.3.0 🎉 

```python
SparseVector([1, 0, 2, 0, 3, 0])
# or
SparseVector({0: 1, 2: 2, 4: 3}, 6)
```
